src/sentry/options/manager.py

```diff
@@ -35,6 +35,15 @@ class UnknownOption(KeyError):
 FLAG_ALLOW_EMPTY = 1 << 6
 # Values that are credentials should not show up in web UI.
 FLAG_CREDENTIAL = 1 << 7
+# Values that are meant to be modified live, eg. for rollout etc.
+FLAG_LIVE_MODIFIABLE = 1 << 8
```

Comment:
Trying to think of another name here, since really all options are modifiable.

Maybe `FLAG_ADMIN_MODIFIABLE` or something? Not totally sure if that's better

Reply:
`FLAG_ADMIN_MODIFIABLE` was literally my other pick, I'll switch it.